forge-gui/res/cardsfolder/upcoming/raze_to_the_ground.txt

```diff
@@ -0,0 +1,8 @@
+Name:Raze to the Ground
+ManaCost:1 R
+Types:Sorcery
+K:This spell can't be countered.
+A:SP$ Destroy | Cost$ 2 G | ValidTgts$ Artifact | SubAbility$ DBDraw | SpellDescription$ Destroy target artifact. If its mana value was 1 or less, draw a card.
+SVar:DBDraw:DB$ Draw | ConditionCheckSVar$ X | ConditionSVarCompare$ LE1
```

Comment:
I think you could use ConditionDefined$ Targeted | ConditionPresent$ Card.cmcLE1 and trim a line

Reply:
@Northmoc it probably need to remember LKI because of clone effects? we had a similar issue before with other stuff

Reply:
Yeah, I think that would be even better